src/Tree/Container.js

```diff
@@ -45,7 +46,8 @@ const Container = (props: Props) => {
     currentTheme,
     showEmptyItems,
     iconSet,
-    noIcons
+    noIcons,
+    expandAll
```

Comment:
I don't think the approach of using a prop is the solution here. It's not very 'react-ful' in its approach. I think a better way (and my preferred approach would be to lift state of all of the nodes up the first tree level, and then create a new 'open' prop on each node, (after making a copy of the initial props).

Reply:
Having reviewed this is full, I think a better way would be to expose some internal 'controls' of the tree as render props, and then a user it free to create whatever logic they want to control the state of the tree inside the render prop. You can use a portal to move the button elsewhere otherwise the controls would be rendered at the bottom of the list. 

additionally, we can 'list' the state up from each node to the top of the tree, to manage selected and expanded states). 

example: https://codesandbox.io/s/exposing-internal-controls-via-render-props-zht0s